packages/example_hardhat/hardhat.config.ts

```diff
@@ -1,21 +1,11 @@
-import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-chai-matchers";
-import "hardhat-deploy";
-import "@nomiclabs/hardhat-ethers";
-import "solidity-coverage";
-import dotenv from "dotenv";
-dotenv.config();
+import '@nomicfoundation/hardhat-toolbox';
+import {HardhatUserConfig} from 'hardhat/config';
 
 const config: HardhatUserConfig = {
-  paths: {
-    sources: "./contracts",
-  },
-  // solidity compiler version may be updated for new packages as required
-  // to ensure packages use up-to-date dependencies
```

Comment:
I see you deleted my comments here, but would like to include some instructions so that future users know they can update the compiler version if they want - would that work? Or do you propose to not change it? @adjisb

Reply:
You are right, I removed the comment by mistake, I will re-add it.